dist/bin/zincati-update-now

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "WARN: This command is experimental and subject to change." >&2
+
+# this should exist already, but in case
+mkdir -p /run/zincati/config.d
```

Comment:
So I am very much out of my depth here but I have to ask in what circumstance would this not exist but the services exist?  
would that be an indication of other issues that might want to halt the update attempt ?

Reply:
doubt it, I think this only means that somehow the [tmpfiles.d fragment](https://github.com/coreos/zincati/blob/f50707131c7f823068c6ef410817b2f064c8424a/dist/tmpfiles.d/zincati.conf#L5) didn't run